tools/ibc-test/Cargo.toml

```diff
@@ -45,7 +45,7 @@ toml_edit = "0.19.14"
 lazy_static = "1.4.0"
 ethers = { version = "2.0.2", features = ["rustls", "ws"] }
 
-forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "64978b97" }
+forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "61a78aa" }
```

Comment:
```suggestion
forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "cd29a30b350ab853a496a58b970017bfa1249182" }
```

Reply:
fix: unable to update https://github.com/synapseweb3/forcerelay-ckb-sdk?rev=61a78aa#61a78aaa

See https://github.com/synapseweb3/forcerelay/actions/runs/6953138151/job/18917831578#step:5:49